Formula/libid3tag.rb

```diff
@@ -66,7 +66,13 @@ class Libid3tag < Formula
     sha256 "5e86270ebb179d82acee686700d203e90f42e82beeed455b0163d8611657d395"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
   def install
+    system "touch NEWS AUTHORS ChangeLog"
```

Comment:
Any reason why this is necessary?

Reply:
```suggestion
    system "touch", "NEWS", "AUTHORS", "ChangeLog"
```

Reply:
without it: 

```
Makefile.am: error: required file './NEWS' not found
Makefile.am: error: required file './AUTHORS' not found
Makefile.am: error: required file './ChangeLog' not found
```

You can sorta around it by doing like:

"AUTOMAKE='automake --foreign' autoreconf" 

but I figured this approach was cleaner on some level